### ISSUE TITLE
About 150 conformations are missing key `mbis_charges`

### ISSUE THREAD

**Issue by leifjacobson** [openmm/spice-dataset#48]
Just an FYI, maybe doesn't need to be an issue.  Most examples missing this key appear to be in the ion pairs set.

**Comment by pavankum** [comment]
@leifjacobson Yeah, you're right. This set doesn't include MBIS charges, https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2022-06-08-QMDataset-ion-pairs#metadata.

**Comment by pavankum** [comment]
There was a high rate of failure in MBIS volume convergence and we decided to exclude it for this set.

**Comment by peastman** [comment]
There's also a small number of DES370K ones that don't, all the ones in the "DES370K Single Points Dataset Supplement" subset.  Those are ones where MBIS failed to converge but all the rest of the calculation worked, so we reran them without MBIS.
